### ISSUE TITLE
[Bug]: AssetController uses fpassthru($asset->getStream()), but it can return null

### ISSUE THREAD

**Issue by dkarlovi** [pimcore/pimcore#13260]
### Expected behavior

Handle null streams correctly

### Actual behavior

If asset stream is null, it produces an error

```
fpassthru(): Argument #1 ($stream) must be of type resource, null given
```

### Steps to reproduce

Have an asset where you cannot obtain its stream and try to download it.

See 

https://github.com/pimcore/pimcore/blob/b2110bb3c416cbf4a7fce967fe98a0e031d642a0/bundles/AdminBundle/src/Controller/Admin/Asset/AssetController.php#L1299
https://github.com/pimcore/pimcore/blob/b2110bb3c416cbf4a7fce967fe98a0e031d642a0/bundles/AdminBundle/src/Controller/Admin/Asset/AssetController.php#L1147

Maybe more.

**Comment by brusch** [comment]
@dkarlovi Thanks for reporting - can you maybe create a PR for that? 😊 Thanks in advance!

**Comment by aryaantony92** [comment]
Fixed by #13279
